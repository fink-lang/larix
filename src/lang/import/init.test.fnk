{describe, it, expect, to_match_snapshot} = import '@fink/jest/test.fnk'

{parse_expr} = import '../../parser.fnk'


describe 'import', fn:
  it 'parses import', fn:
    expect
      parse_expr "import './foobar.fnk'"
      to_match_snapshot
