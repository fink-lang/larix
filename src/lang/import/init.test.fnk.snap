// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`import parses import 1`] = `
prefix import (1:0-1:21)
  string ' (1:7-1:21)
    './foobar.fnk'
`;
